tests/integration/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+# SPDX-License-Identifier: GPL-2.0
```

Comment:
`GPL-2.0` is ambiguous -- you want either `GPL-2.0-or-later` or `GPL-2.0-only`.

Reply:
This is a leftover from `bpfilter`'s kernel era. According to [this documentation](https://www.kernel.org/doc/html/v6.4/process/license-rules.html) the proper tag should then be `GPL-2.0-only`. I'll update this one in this PR and put up another PR to change it on the whole project.